### ISSUE TITLE
https://github.com/SynoCommunity/spksrc/wiki/Developers-HOW-TO : "Build a package" Tutorial does not work

### ISSUE THREAD

**Issue by exse2** [SynoCommunity/spksrc#4122]
mkdir spksrc
cd spksrc
git clone https://github.com/SynoCommunity/spksrc.git
sudo yum install python2-demjson
make setup
cd spk/transmission/
make clean

make arch-braswell #**fails now**

#FIX:
Edit ../../toolchains/syno-braswell-6.1/Makefile
        TC_DIST_SITE is wrong. It's /files/Tool%20Chain/ instead of just /files/

make arch-braswell #**fails again**

#Fails with "Expecting property name: line 4 column 145 (char 233)"

#FIX: 
vi work-braswell-6.1/staging/app/config
        "desc" is malformed. In ../../mk/spksrc.service.mk change '@echo "\",\n    \"icon\":' into '@echo "\",    \"icon\":'

make arch-braswell #**works now**

And while we're at it, "git clone https://You@github.com/You/spksrc.git ~/spksrc" described in "https://github.com/SynoCommunity/spksrc"->"Setup Development Environment"->"Docker" also does not work. It needs a password I don't have.

**Comment by publicarray** [comment]
As for the wrong toolchain URL this is tracked at #4115

**Comment by publicarray** [comment]
The git clone is refereeing to a fork that you can make and If you haven’t configured /installed SSH than yes it won’t work. Maybe we should change this to use https instead since it’s almost guaranteed to work everywhere.

**Comment by publicarray** [comment]
Closing as this addressed in 6c5e190e004625b8c86346e3a4d973dd98da9596 and #4122 +verified locally that building transmission works without any errors

The JSON that I got is valid:

```
{ ".url": { 
  "com.synocommunity.packages.transmission": {
    "title": "Transmission",
    "desc": "Transmission is an easy and fast BitTorrent client. You can control it remotely with its web interface or dedicated applications.",
    "icon": "images/transmission-{0}.png",
    "type": "url",
    "protocol": "http",
    "port": "9091",
    "url": "/",
    "allUsers": true,
    "grantPrivilege": "all",
    "advanceGrantPrivilege": true
} } }

```
